app/browser/api/ledger.js

```diff
@@ -81,6 +83,7 @@ let verifiedTimeoutId = false
 let v2RulesetDB
 const v2RulesetPath = 'ledger-rulesV2.leveldb'
 const statePath = 'ledger-state.json'
+const referralCodePath = 'promoCode/code'
```

Comment:
Please update to simply 'promoCode', or even better use the function for reading first-run promoCode from the module I included in #12920

Reply:
will change to your read when #12920 is merged

Reply:
done